packages/dialect-react-sdk/hooks/useGlobalUnreadMessages.ts

```diff
@@ -0,0 +1,31 @@
+import useDialectSdk from './useDialectSdk';
+import useSWR from 'swr';
+import { CACHE_KEY_THREADS_SUMMARY } from './internal/swrCache';
+import { EMPTY_OBJ } from '../utils';
+
+interface UseGlobalUnreadMessagesParams {
+  refreshInterval?: number;
+}
+
+interface UseGlobalUnreadMessageValue {
+  unreadCount: number;
+  refresh: () => void;
+}
+
+const useGlobalUnreadMessages = ({
```

Comment:
maybe we could keep only one hook for this?
e.g. keep the logic in useUnreadMessages hook depending if public key is passed or not?

I'm worrying about it, cause we have a ton of public hooks already 😅

Reply:
I hear you. It did seem like an unnecessary hook, given one already exists.

I initially followed the logic I used in data-service, that this seems like a separate thing, since it's "global", across all dialects. Let me try to merge them together, but I feel like we're bloating one hook.

Reply:
Done, sir!